src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -2485,7 +2485,6 @@ public PreviewSummary declaredUsingPreviewAPIs(Element el) {
                     usedInDeclaration.addAll(types2Classes(tpe.getBounds()));
                 }
                 usedInDeclaration.addAll(types2Classes(List.of(te.getSuperclass())));
-                usedInDeclaration.addAll(types2Classes(te.getInterfaces()));
                 usedInDeclaration.addAll(types2Classes(te.getPermittedSubclasses()));
```

Comment:
This one is dubious too - imagine if we have a classfile `Attribute` that is preview-only, such as `Pattern` or `LoadableDescriptors`.  But that is out of scope for this patch.